test/verify/check-connection

```diff
@@ -1109,6 +1109,43 @@ UnixPath=/run/cockpit/session
         m.start_cockpit()
         self.login_and_go("/system")
 
+    @skipOstree("test assumes local install")
+    @nondestructive
+    def testXdgConfig(self):
+        m = self.machine
+        self.write_file("/etc/systemd/system/service.d/xdg.conf",
+                        "[Service]\nEnvironment=XDG_CONFIG_DIRS=/var/tmp/xdg:/etc/test-xdg\n",
```

Comment:
An empty value actually means "*do* erase the ones that came before" (i.e. "reset"). I explicitly don't want that, it would destroy services (including cockpit.service). This is meant to be an addition.